gateway/sds_gateway/users/views.py

```diff
@@ -1462,8 +1636,12 @@ def get(self, request, *args, **kwargs) -> JsonResponse:
             return JsonResponse(response_data)
 
         except (ValueError, TypeError) as e:
-            log.warning("Invalid parameter in captures API request: %s", e)
-            return JsonResponse({"error": "Invalid search parameters"}, status=400)
+            import traceback
+            error_msg = str(e)
+            traceback_str = traceback.format_exc()
+            log.error(f"Invalid parameter in captures API request: {error_msg}")
+            log.error(f"Full traceback:\n{traceback_str}")
+            return JsonResponse({"error": f"Invalid search parameters: {error_msg}"}, status=400)
```

Comment:
keep .warning, use .exception if you want the trace; don't use .error to log 4xx errors.